### DISCUSSION TITLE
is it possible to deploy slurm-web agent on different node with slurmrestd?

### DISCUSSION THREAD

**Discussion by csskysea** [rackslab/Slurm-web#370] [Q&A] [answered]
is it possible to deploy slurm-web agent on different node with slurmrestd? if it is, what should i do?

**Comment by rezib** [comment, marked as answer]
Hello @csskysea, I converted your issue #369 into a discussion as it is more a question that an actual issue.

For the moment, Slurm-web agent must run on the same node as slurmrestd as it only support connection over Unix socket. For reference, it is explained in [architecture documentation](https://docs.rackslab.io/slurm-web/overview/architecture.html#protocols).

This limitation may be removed when https://github.com/rackslab/Slurm-web/issues/313 is implemented.
